Tomas Heran <tomas.heran@oracle.com>
libnsl and libsocket need to be linked to on Solaris 11 for socket API.
Reported upstream as: https://gnunet.org/bugs/view.php?id=3752

diff -r d118e0f483a4 configure.ac
--- a/configure.ac	Thu Feb 05 14:49:09 2015 +0100
+++ b/configure.ac	Thu Apr 09 16:39:50 2015 +0200
@@ -135,6 +135,8 @@
 *solaris*)
      AC_DEFINE_UNQUOTED(SOLARIS,1,[This is a Solaris system])
      AC_DEFINE_UNQUOTED(_REENTRANT,1,[Need with solaris or errno doesnt work])
+     AC_SEARCH_LIBS(gethostbyname, nsl)
+     AC_SEARCH_LIBS(socket, socket)
      ;;
 *arm-linux*)
      AC_DEFINE_UNQUOTED(LINUX,1,[This is a Linux kernel])
